Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -100,6 +121,11 @@ void ITSTrackTask::monitorData(o2::framework::ProcessingContext& ctx)
     std::map<std::string, std::string> metadata;
     mDict = TaskInterface::retrieveConditionAny<o2::itsmft::TopologyDictionary>("ITS/Calib/ClusterDictionary", metadata, ts);
     ILOG(Debug, Devel) << "Dictionary size: " << mDict->getSize() << ENDM;
+
+    o2::its::GeometryTGeo::adopt(TaskInterface::retrieveConditionAny<o2::its::GeometryTGeo>("ITS/Config/Geometry", metadata, ts));
```

Comment:
If we do not use geometry in the code besides alignment monitoring, could you make geometry downloading optional?

Reply:
It is implemented in the code and failure case(throw) to download ccdb object for ITS geometry is added.